connector/src/main/java/org/geysermc/connector/entity/AbstractArrowEntity.java

```diff
@@ -49,10 +51,15 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
     }
 
     @Override
-    public Vector3f getBedrockRotation() {
+    public void setRotation(Vector3f rotation) {}
```

Comment:
Is there any point where we don't want to ignore the server's arrow value?

Reply:
No, as the Java client ignores the server's rotation value too. It calculates the rotation from its motion every tick.